frontend/src/components/ResultPopUp/ResultPopUp.jsx

```diff
@@ -2,28 +2,51 @@ import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import "./ResultPopUp.css";
 
-function ResultPopUp({ gameOver, score }) {
+function ResultPopUp({ score, questionIndex, setQuestionIndex, questionOver }) {
+  const handleClick = () => {
+    setQuestionIndex(questionIndex + 1);
+  };
+
   return (
-    gameOver && (
+    questionOver && (
       <section className="popUp">
-        <h2 className="popUp__title">Fin de partie</h2>
+        <h2 className="popUp__title">
+          {questionIndex < 10 ? `Film ${questionIndex}/10` : `Fin de partie`}
+        </h2>
         <p className="popUp__paragraph">Votre score est : {score} points</p>
         <nav className="popUp__buttons">
-          <Link className="popUp__button" to="/">
-            Menu
-          </Link>
-          <Link className="popUp__button" to="/ranking">
-            Classement
-          </Link>
+          {questionIndex < 10 ? (
+            <button
+              type="button"
+              className="popUp__button"
+              onClick={handleClick}
+            >
+              Continuer
+            </button>
+          ) : (
+            <>
+              <Link className="popUp__button popUp__button--first-child" to="/">
+                Menu
+              </Link>
+              <Link
+                className="popUp__button popUp__button--last-child"
```

Comment:
It's ninja code, too close to a pseudo-element. You could just use the right pseudo-element rather than a class with a misleading name and a source of future bugs.